### PR TITLE
Add and fill out changelog

### PR DESCRIPTION
**Pull request by AnEmortalKid**
add changelog

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
+- '10.21.0'
 - '8.8.1'
```
Comment (AnEmortalKid) [inline .travis.yml:4]: @Pholey node 6 got EOL'd.......... figured it isn't worth it trying to wrangle deps anyway

Comment (Pholey) [reply .travis.yml:4]: @AnEmortalKid sounds good to me

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+- Loading of dogescript files in REPL [#111](https://github.com/dogescript/dogescript/pull/111)
```
Comment (Pholey) [inline CHANGELOG.md:10]: Love this, what do you think about using tags and https://github.com/dogescript/dogescript/releases/new for this kind of documentation?

Comment (AnEmortalKid) [reply CHANGELOG.md:10]: I totes forgot that was a thing. I think if I tag things correctly with a `milestone` and `enhancement` or `bug` we can then auto-generate the content for that. 

Lemme look at some samples cause i've seen some projects with good notes with 🐛  📦  🚀  style icons for each item type.

**Pull request merged**